### PR TITLE
Clear eslint warnings and clean up code

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
I neatened the whitespace for the files in ``` src -> components -> React```. The other files were too long to justify cleaning everything up right now. No changes to code functionality on the branch.

There's still some warnings with React but I haven't fully updated the dependency yet, will focus all the react issues to one branch.

### CODE REVIEW

**Comment by kmilo9999** [comment]
Good Job !!

Also, Make sure it compiles and all the functionalities persist after your changes. On every iteration you should be clearing the cache and load the app from scratch. Just to make sure it works properly.
What browser are you testing with?  I basically tested only on FireFox because at the time it was the only browser that supported WebGL 2.0.

File: src/libs/three.module.js [hunk 1]
```diff
@@ -30663,11 +30663,11 @@ function SphereBufferGeometry( radius, widthSegments, heightSegments, phiStart,
 
 		var uOffset = 0;
 
-		if ( iy == 0 && thetaStart == 0 ) {
```
Comment (kmilo9999) [inline src/libs/three.module.js]: This line  was producing warnings, correct? I dont think you should modify this file, it is the "three js" devs job to maintain it.

**Comment by RobertGemmaJr** [comment]
I've been using chrome but I can switch to Firefox.

By cache do you just mean the ```node_modules/``` and ```package-lock.json```? I've been deleting them and reinstalling to test if it works.

I can check if WebGl 2.0 is supported on other browsers now too

**Comment by kmilo9999** [comment]
I meant the browser cache. Just as sanity check.
Yes, make sure to test it on browsers that support web-GL 2.  Probably, by now all of them do. That wasnt the case 2 years ago.

File: src/libs/three.module.js (hunk 1 above)
Comment (RobertGemmaJr) [reply src/libs/three.module.js]: Yes and I did fix it, should I revert that commit?

Comment (kmilo9999) [reply src/libs/three.module.js]: Yeah. Let this file intact. The change in there other ones are fine !!

Good work !

**Comment by RobertGemmaJr** [comment]
Yeah it looks like most browser now support it: https://caniuse.com/webgl2, just checked Chrome, Firefox, and Opera on my laptop. Not sure about VR devices though

**Comment by kmilo9999** [comment]
Those are enough!
Give it a a try with the quest2.

**Review by kmilo9999 — APPROVED** [review]

**Pull request merged**